packages/colossus/lib/sync.js

```diff
@@ -100,7 +100,7 @@ async function sync_periodic(api, config, storage)
 
 function start_syncing(api, config, storage)
 {
-  setTimeout(sync_periodic, config.get('syncPeriod'), api, config, storage);
+  sync_periodic(api, config, storage);
```

Comment:
Just so I understand, this only applies to when you first start the server? Isn't there a lot of other processes that needs to be completed before starting syncing content anyway?

Reply:
Correct this is the first call when starting up after checking the node is staked, storage initialized etc.. here: https://github.com/Joystream/storage-node-joystream/blob/development/packages/colossus/bin/cli.js#L328

Reply:
Ok, so it just removes the extra 30sec.